recipes-extended/libcobalt/libcobalt_git.bb

```diff
@@ -0,0 +1,65 @@
+LICENSE = "BSD-3-Clause & BSD-2-Clause & Apache-2.0 & MIT & ISC & OpenSSL & CC0-1.0 & LGPL-2.0 & LGPL-2.1 & PD & Zlib & MPL-2.0"
+LIC_FILES_CHKSUM = " \
+    file://src/LICENSE;md5=0fca02217a5d49a14dfe2d11837bb34d \
+"
+
+PR = "r0"
+PACKAGES = "${PN}"
+DEPENDS = "gstreamer1.0 gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad ninja-native bison-native wpeframework"
+
+SRC_URI = "git://git@github.com/Metrological/Cobalt.git;protocol=https;rev=c0e5376052e4c99e9e99866198e78fcff61d42f3 \
+           git://chromium.googlesource.com/chromium/tools/depot_tools.git;protocol=https;rev=44ea3ffa40fe375a8ae2e0f89968c335d08c8a8a;destsuffix=depot_tools;name=depot_tools \
+"
+S = "${WORKDIR}/git"
+
+LD = "${CXX}"
+
+# TODO: we might also have mips here at some point.
+COBALT_PLATFORM ?= ""
+COBALT_PLATFORM_brcm ?= "wpe-brcm-arm"
+COBALT_PLATFORM_rpi ?= "wpe-rpi"
+
+COBALT_DEPENDENCIES ?= ""
+COBALT_DEPENDENCIES_brcm ?= "gst1-bcm"
```

Comment:
no gst-gl dependency in case of RPI?

Reply:
Yes all are in the generic DEPENDS part. This I missed to ad to DEPENDS, now added.

Reply:
We might need to set the right RDEPENDS then for individual plugins that would be required by Cobalt. As that would ultimately drive the include of the right plugins on the target filesystem.

Similar to WPEWebKit: https://github.com/WebPlatformForEmbedded/meta-wpe/blob/morty/recipes-wpe/wpewebkit/wpewebkit.inc#L137

I think you're fine for now as long as WPEWebKit already includes the super set of what Cobalt might need. But if you would ever make a cobalt only image that might cause some trouble.